tests/gnmi/conftest.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import shutil
+
+from tests.common.helpers.assertions import pytest_require as pyrequire
+from tests.common.helpers.dut_utils import check_container_state
+from helper import GNMI_CONTAINER_NAME, apply_cert_config, create_ext_conf
+from tests.generic_config_updater.gu_utils import create_checkpoint, rollback
+
+SETUP_ENV_CP = "test_setup_checkpoint"
+
+
+@pytest.fixture(scope="function", autouse=True)
+def skip_non_x86_platform(duthosts, rand_one_dut_hostname):
+    """
+    Skip the current test if DUT is not x86_64 platform.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    platform = duthost.facts["platform"]
+    if 'x86_64' not in platform:
+        pytest.skip("Test not supported for current platform. Skipping the test")
+
+
+@pytest.fixture(scope="module", autouse=True)
+def download_gnmi_client(duthosts, rand_one_dut_hostname, localhost):
```

Comment:
>download_gnmi_client [](http://example.com/codeflow?start=4&length=20)

telemetry is tested with ptf "gnmi_cli_py/py_gnmicli.py". Is it better than download_gnmi_client?

Reply:
I need GNOI client, but py_gnmicli.py only supports GNMI.